editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -4929,8 +4929,8 @@ void VisualShaderEditor::_show_preview_text() {
 	preview_showed = !preview_showed;
 	if (preview_showed) {
 		if (preview_first) {
-			preview_window->set_size(Size2(400 * EDSCALE, 600 * EDSCALE));
 			preview_window->popup_centered();
+			preview_window->set_size(Size2(400 * EDSCALE, 600 * EDSCALE));
```

Comment:
That's weird that this would be the fix. Sounds like there's a deeper issue if it's not working to set the size before showing the window.

Isn't this a recent regression? Might be worth finding what's the culprit before working it around.
CC @Sauermann maybe

Reply:
I believe, that this change will center the top-left corner of the window, but not the window itself.